salt/modules/container_resource.py

```diff
@@ -123,11 +124,18 @@ def run(name,
         python_shell=True,
         output_loglevel='debug',
         ignore_retcode=False,
+        path=None,
         use_vt=False,
         keep_env=None):
     '''
     Common logic for running shell commands in containers
 
+    path
+        path to the container parent (for LXC only)
+        default: /var/lib/lxc (system default)
+
+        .. versionadded:: 2015.5.0
```

Comment:
s/2015.5.0/Beryllium/

Since this in in develop, it will be in the Beryllium release. Which will probably be 2015.8, but that's not set in stone.